Makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: build-local build-noop-dest run-local run-latest run-latest-nightly print-results
 
+build:
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build main.go
```

Comment:
Hmm, why are we building for linux here? I understand it for the plugin as we expect it to run inside a docker container, but this tool will run on the host, right?

Reply:
Good point. I've copy pasted it from the other command, but it's current usage is only in the GitHub workflow, which is on Linux (to make sure it builds). I'll fix.